### DISCUSSION TITLE
Long term support policy

### DISCUSSION THREAD

**Discussion by itoshun** [vercel/next.js#41313] [Help]
I am wondering what is the Long term support policy for major versions.
For example, version 10.0 was released on October 27th 2020, and until when do you keep solving bugs on version 9 ?

**Comment by CatiaGomes82** [comment]
Would also be interested in knowing this. I cannot find any documentation about it.

**Comment by johnavery** [comment]
I am also interested in this information, for enterprise adoption.  What is Vercel's long-term support policy for next.js, by version?

**Comment by mluypaert** [comment]
+1, especially relevant with the release of Next.js v15 and its requirement to upgrade to react v19. How long will we get support and patches for Next.js v14?
